install.sh

```diff
@@ -102,15 +109,17 @@ fi
 
 if [[ "$ARCH" == "x86_64" ]]; then
   LIB_SUFFIX='64'
+else
+  LIB_SUFFIX=''
 fi
 
 # Warn users of the AMD segfault issue and allow them to work around it.
 # The easiest way to distinguish StoneyRidge platorms is to check for the FMA4
 # instruction, as it was first introduced in Bulldozer and later dropped in Zen.
-if grep -s "fma4" /proc/cpuinfo ; then
+if grep -q "fma4" /proc/cpuinfo ; then
   echo_info "Notice: You are running an AMD StoneyRidge device; due to some bugs some packages may fail with a segmentation fault and need to be rebuilt."
-  echo_info "If this happens, please report them to: https://github.com/chromebrew/chromebrew/issues"
-  echo_info "If the install fails, try running 'CREW_AMD_INSTALL=1 exec bash --init-file <(curl -Ls git.io/vddgY)'"
+  echo_info "If this happens, please report them to: ${REPO_URL}/issues"
+  echo_info "If the install fails, try running 'CREW_AMD_INSTALL=1 exec bash --init-file <(curl -L git.io/vddgY)'"
```

Comment:
Now you are starting to think like me.  I don't like when ANY command is silenced.